gen/flights/traveltime.py

```diff
@@ -72,16 +72,24 @@ def mission_start_time(flight: Flight) -> timedelta:
         return startup_time
 
     def earliest_tot(self) -> timedelta:
-        earliest_tot = max(
```

Comment:
I think I prefer keeping everything as-is and adding:

```
if not self.package.flights:
    return timedelta(0)
```

to the start of the function.